ios/app/brave_core_main.mm

```diff
@@ -215,7 +215,7 @@ - (instancetype)initWithUserAgent:(NSString*)userAgent
     ios::ChromeBrowserStateManager* browserStateManager =
         GetApplicationContext()->GetChromeBrowserStateManager();
     ChromeBrowserState* chromeBrowserState =
-        browserStateManager->GetLastUsedBrowserState();
+        browserStateManager->GetLastUsedBrowserStateDeprecatedDoNotUse();
```

Comment:
Do we need to create a follow up issue to rework these to whatever the new method is?

Reply:
Created https://github.com/brave/brave-browser/issues/38419 to track this. cc: @kylehickinson